examples/chain_request_resubmission.py

```diff
@@ -0,0 +1,59 @@
+"""
+This examples shows some alternatives to patch several chained
+requests when it is required to apply changes on its
+the chained campaign.
+"""
+
+import random
+from datetime import datetime
+
+from rest import McM
+from rest.applications.mcm.resubmission import (
+    ChainRequestResubmitter,
+    InvalidateDeleteRequests,
+)
+
+# Instances for the client.
+mcm = McM(id=McM.OIDC, dev=True)
+chain_resubmitter = ChainRequestResubmitter(mcm=mcm)
+invalidator = InvalidateDeleteRequests(mcm=mcm)
+
+# For this example, let's modify the block number for a chained campaign
+# and resubmit all the chained request linked to it by using the related
+# root request.
+chained_campaign_prepid = "chain_Run3Summer22EEFSGenPremix_flowRun3Summer22EEFSMiniAODv4_flowRun3Summer22EEFSNanoAODv12"
+chained_campaign = mcm.get(
+    object_type="chained_campaigns", object_id=chained_campaign_prepid
+)
+assert isinstance(chained_campaign, dict), "Chained campaign not found"
+
+block_number = random.randint(1, 6)
```

Comment:
Please don't leave this here, people will forget to update it and end up with chains randomly scattered across blocks

Reply:
Okay, I will set a specific block.